e2e-tests/main_test.go

```diff
@@ -0,0 +1,114 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"os/exec"
+	"strconv"
+	"testing"
+	"time"
+)
+
+func runCommandWithOutput(cmd *exec.Cmd) (string, error) {
+	var stdout, stderr bytes.Buffer
+	cmd.Stdout = &stdout
+	cmd.Stderr = &stderr
+
+	err := cmd.Run()
+	if err != nil {
+		return "", fmt.Errorf("error running command: %v\nstdout: %s\nstderr: %s", err, stdout.String(), stderr.String())
+	}
+
+	return stdout.String(), nil
+}
+
+// From the output of the AbciInfo command, extract the latest block height.
+// The json bytes should look e.g. like this:
+// {"jsonrpc":"2.0","id":1,"result":{"response":{"data":"interchain-security-p","last_block_height":"2566","last_block_app_hash":"R4Q3Si7+t7TIidl2oTHcQRDNEz+lP0IDWhU5OI89psg="}}}
```

Comment:
nit: In the provided [doc](https://github.com/cometbft/cometbft/blob/v0.37.x/spec/abci/abci++_methods.md#info there are more fields in a response (e.g., `app_version`). I guess we do not care about those?

Reply:
We could, but I want to keep this general to be as easy to maintain as possible, so don't want to test for specifics so much.
This is more high-level to have some base-level confidence that things are working in some shape. But thanks for pointing this out!